From 9e15f8af7ce0fef4d588ffdceb1adea9e6f4b57c Mon Sep 17 00:00:00 2001
From: Lukas Czerner <lczerner@redhat.com>
Date: Fri, 21 Jun 2013 16:42:27 +0200
Subject: [PATCH 5/8] mke2fs.conf: Enable 64bit feature by default

The 64bit feature has been added to the ext4 a long time ago (2006?) so
it should be ok to enable it by default by now. This would allow us to
resize the file system past 16TB when it was originally smaller than
that and user did not specified 64bit feature manually.

Signed-off-by: Lukas Czerner <lczerner@redhat.com>
---
 misc/mke2fs.conf.in | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/misc/mke2fs.conf.in b/misc/mke2fs.conf.in
index 0871f77..667800c 100644
--- a/misc/mke2fs.conf.in
+++ b/misc/mke2fs.conf.in
@@ -11,8 +11,7 @@
 		features = has_journal
 	}
 	ext4 = {
-		features = has_journal,extent,huge_file,flex_bg,uninit_bg,dir_nlink,extra_isize
-		auto_64-bit_support = 1
+		features = has_journal,extent,huge_file,flex_bg,uninit_bg,dir_nlink,extra_isize,64bit
 		inode_size = 256
 	}
 	ext4dev = {
-- 
1.8.3.1

